src/blockchain_txn_payment_v2.proto

```diff
@@ -1,12 +1,15 @@
 syntax = "proto3";
 package helium;
 
+import "blockchain_token_type_v1.proto";
+
 message blockchain_txn_payment_v2 {
   bytes payer = 1;
   repeated payment payments = 2;
   uint64 fee = 3;
   uint64 nonce = 4;
   bytes signature = 5;
```

Comment:
So we think payment txns will always have to be one or more non-zero payments of the same token type, rather than adding a token_type to the payment record and being able to mix and match?

Reply:
yeah, I don't think we want to mix and match in the individual payment record. Probably best to just have top level token_type specified in the payment txn itself. Unless there's some really compelling reason to do so. Mostly just saying to keep things clean-ish txn level

Reply:
The one edge-case I can think of would be balance-clearing txns. If the intent for those is to just clear a single token-type that should work but if the intent is to “burn out my account” then we’d need to generate an enumerable higher-level operation to create a clearing txn for each held token type

Reply:
I think the token type should be on the individual payment and you should be able to transfer multiple.

Reply:
I agree with @abhay here

Reply:
Hmm, alright then. I'll tweak the proto to reflect that individual payment has the token type instead.